### PR TITLE
KEY-502: Bring extensions-cli v1.0 and master (v1.1+) to parity

### PR DESCRIPTION
**Pull request by zxan1285**
## ✏️ Changes
  `extensions-cli` is using the most recent versions of babel and webpack modules. This leads to conflict with client building process of our extensions. It these changes i added client:build command (`a0-ext build:client ./client/app.jsx ./dist`) and `devServer`, as client webtask config and webtask-dev-server options are the same for all extensions.
  
## 🔗 References
  Jira: https://auth0team.atlassian.net/browse/KEY-502
  
## 🎯 Testing
✅ This change has been tested locally

### CODE REVIEW

File: lib/dev-server.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      poll: 1000
+    },
+
+    stats: { colors: true },
+    headers: {
+      'Access-Control-Allow-Origin': '*'
+    }
+  };
+
+  new WebpackDevServer(compiler, options)
```
Comment (FadyMak) [inline lib/dev-server.js]: Why do we need a dev server in the build step?

File: bin/a0-ext [hunk 2]
```diff
@@ -52,5 +54,31 @@ prog
             });
           });
       });
+  })
+  .command('build:client', 'Compile your React Frontend into an Auth0 Extension Client')
```
Comment (FadyMak) [inline bin/a0-ext]: Will all extensions have to be updated to have a `client:build` step? We need to make sure this fits into the CI/CD plans cc @shawnmclean

Comment (FadyMak) [inline bin/a0-ext]: Will all extensions need to be updated with a `build:client` command? If so I'd like to make sure it fits into @shawnmclean's plans for CI/CD

File: lib/client-config.js [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+            priority: 10,
+            enforce: true
+          }
+        }
+      }
+    }
+  };
+
+  const extractCSS = () => {
+    const cssRule = config.module.rules.find(rule => rule.test.test('foo.css'));
```
[the remark this answers is not in the document]
Comment (zxan1285) [reply lib/client-config.js]: No. That's one way to find css rule to add `mini-css-extract-plugin`.

File: bin/a0-ext (hunk 2 above)
Comment (zxan1285) [reply bin/a0-ext]: Yes, but it's just `"client:build": "a0-ext build:client ./client/app.jsx ./dist",` vs old `"client:build": "cross-env NODE_ENV=production webpack --config ./build/webpack/config.prod.js --colors -p",`

File: lib/dev-server.js (hunk 1 above)
Comment (zxan1285) [reply lib/dev-server.js]: It isn't a build step. The only reason why i added dev-server here is that we already have client webpack config and dependencies in `extensions-cli` (and dev-server options are the same for all extensions).

**Pull request merged**